apps/ledger-live-mobile/src/components/Web3AppWebview/helpers.ts

```diff
@@ -424,10 +424,12 @@ function useUiHook(): UiHook {
         });
       },
       "storage.get": async ({ key, storeId }) => {
-        return (await deviceStorage.get(`${storeId}-${key}`)) as string;
+        const data = await storage.get<string>(`${storeId}-${key}`);
+        if (data == null || Array.isArray(data)) return "";
+        return data;
```

Comment:
```suggestion
       "storage.get": async ({ key, storeId }): Promise<string> => {
  const value = await storage.get(`${storeId}-${key}`);
  return typeof value === "string" ? value : "";
},
```

Something like that shouldn't be better?

Reply:
Yep ! Nice catch